project/src/main/java/com/google/sps/classes/EmailSender.java

```diff
@@ -0,0 +1,34 @@
+package com.google.sps.classes;
+import java.util.*;
+import javax.mail.*;
+import javax.mail.internet.*;
+import javax.activation.*;
+  
+  
+public class EmailSender{ 
+   public static void sendEmail(String to) {
+      String from = "christinachance315@gmail.com";
+      String host = "localhost";
+      Properties properties = System.getProperties();
+      properties.setProperty("mail.smtp.host", host);
+      Session session = Session.getDefaultInstance(properties);
+
+      try {
+         MimeMessage message = new MimeMessage(session);
+
+         message.setFrom(new InternetAddress(from));
+
+         message.addRecipient(Message.RecipientType.TO, new InternetAddress(to));
+
+         message.setSubject("[Covidiena] Account Verification");
+
+         message.setContent("<h1>An account on Covidiena has vbeen created with this email. To verify this is you, please click <a href='covidiena.appspot.com/login.html'here</h1>.", "text/html");
```

Comment:
maybe separate the content as a string , there is a multiple ways to build a multiple line string to make this cleaner. example see https://blog.softhints.com/java-8-multiline-string-example/

Reply:
I tried doing this but the content isn't a string it is an object of type text/html, so it isn't read if I declare the content earlier.

Reply:
It may cause by the missing closing </a>

Can you try something like: 
```
StringBuilder htmlBuilder = new StringBuilder();
htmlBuilder.append("<h1>");
htmlBuilder.append("An account on Covidiena has vbeen created with this email. To verify this is you, please click");
htmlBuilder.append("<a href='covidiena.appspot.com/login.html'>here</a>");
htmlBuilder.append("</h1>");
String html = htmlBuilder.toString();
```
This will give you a clear view of the pair of open/close tags.

Reply:
done.

Reply:
If this doesn't work, please add closing tag `</a>` to the string.

Reply:
It works now